Airport Ticket Booking System/Services/ManagerService.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Airport_Ticket_Booking_System;
+
+public partial class ManagerService
+{
+    private readonly FlightService _flightService;
+    private readonly IBookingService _bookingService;
+    private readonly IBookingRepository _bookingRepository;
+    private readonly FlightRepository _flightRepository;
+
+
+    public ManagerService(FlightService flightService, IBookingService bookingService, IBookingRepository bookingRepository, FlightRepository flightRepository)
```

Comment:
seems like this won't be needed anymore? Lets break it down to multiple services as mentioned above.
A side note, one big sign that a service violates the SRP is having many construction injections/params and they are not really related to each other much.